shopinvader_api_sale_loyalty/routers/cart.py

```diff
@@ -0,0 +1,207 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from typing import Annotated
+from uuid import UUID
+
+from fastapi import Depends
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+from odoo.addons.base.models.res_partner import Partner as ResPartner
+from odoo.addons.fastapi.dependencies import (
+    authenticated_partner,
+    authenticated_partner_env,
+)
+from odoo.addons.sale.models.sale_order import SaleOrder
+from odoo.addons.shopinvader_api_cart.routers import cart_router
+from odoo.addons.shopinvader_api_cart.schemas import CartTransaction
+
+from ..schemas import LoyaltyCardInput, LoyaltyRewardInput, Sale
+
+
+@cart_router.post("/apply_coupon", deprecated=True)
+@cart_router.post("/apply_coupon/{cart_uuid}", deprecated=True)
+@cart_router.post("/coupon")
+@cart_router.post("/{cart_uuid}/coupon")
```

Comment:
@marielejeune  can you rename cart_uuid to uuid (or change uuid to cart_uuid) in the base cart module, for consistency and a nicer display in swagger?

Reply:
Done in a new commit